src/components/HistoryTables/MessageHistory/History/MessageConfirmedRow.tsx

```diff
@@ -45,14 +48,12 @@ export default function MessageHistoryRow(props: MessageHistoryRowProps) {
           hideCopy
         />
       </td>
-      {props.inspectingAddress && (
-        <td>
-          <Badge
-            color={toAddressIsInspecting ? 'green' : 'yellow'}
-            text={toAddressIsInspecting ? 'IN' : 'OUT'}
-          />
-        </td>
-      )}
+      <td>
```

Comment:
Fine taking this out, but what prompted this change? Why did it occur in this PR?

Reply:
I think I accidentally came across it :) either way the conditional doesn't seem to be necessary, especially now `inspectingAddress` is a reqiured prop. Also if it _would_ be conditional, we `td` should still be rendered to prevent a column shift